src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -71,20 +73,21 @@ const ChatBtn = styled.img`
 const HeaderTitle = styled.div`
   display: flex;
   text-align: left;
-  margin: 0 auto;
-  font-weight: 900;
-  line-height: 17px;
+  justify-content: center ;
+  font-weight: 700;
+  line-height: 57px;
```

Comment:
It's too much, the text is so separated of subtitle how @angelik0828 has commented. Please consider to use a lower number.

On the another hand you can to use `line-height` values instead of PX. `line-height` uses `em` for default, instead of `px`. Then you could to use `line-height: 1.7` instead of `line-height: 57px`

![imagen](https://user-images.githubusercontent.com/66505715/160883646-73ffc4c1-8af0-453d-8100-cf1cf9108a6a.png)

As you see, it's showing the same result that using 57px, and the number is not so high

![imagen](https://user-images.githubusercontent.com/66505715/160883744-6ae1d14e-c305-4705-b678-f159fba5f448.png)